### PR TITLE
Implement rskip 203

### PR DESCRIPTION
**Pull request by rmoreliovlabs**
## Description
Implement new Environment contract as per described in  RSKIP-203.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Review by Vovchyk — CHANGES_REQUESTED** [review]

**Review by asoto-iov — APPROVED** [review]

**Comment by nagarev** [comment]
pipeline:run

**Review by Vovchyk — DISMISSED** [review]

**Comment by Vovchyk** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

File: rskj-core/src/main/resources/reference.conf [hunk 1]
```diff
@@ -53,6 +53,7 @@ blockchain = {
             rskip199 = iris300
             rskip200 = iris300
             rskip201 = iris300
+            rskip203 = arrowhead600
```
Comment (marcos-iov) [inline rskj-core/src/main/resources/reference.conf:56]: Could you also add this RSKIP in ActivationConfigsForTest::getArrowhead600Rskips?

**Review by marcos-iov — DISMISSED** [review]

File: rskj-core/src/main/resources/reference.conf (hunk 1 above)
Comment (rmoreliovlabs) [reply rskj-core/src/main/resources/reference.conf:56]: Done

**Review by marcos-iov — DISMISSED** [review]

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by rmoreliovlabs** [comment]
pipeline:run

**Comment by Vovchyk** [comment]
pipeline:run

**Review by Vovchyk — APPROVED** [review]

**Pull request merged**